File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums)<=1:
            return len(nums)
        index = 0
        i ,k = 0,1
        while k < len(nums):
            if nums[i] != nums[k]:
                nums[index] = nums[k-1]
                index  += 1
                i = k
            k += 1
            if k == len(nums):
                nums[index] = nums[k-1]
                index +=1
        return index

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        

        int left = 0;
        for(int right = 0;right<nums.length;right++){

            nums[left++] = nums[right];

            while(right < nums.length-1 && nums[right] == nums[right+1])right++;
        }
        return left;
    }
}